### DISCUSSION TITLE
Can't bind open property of details element

### DISCUSSION THREAD

**Discussion by GeorgeFlorian** [alpinejs/alpine#3979] [7. Bugs (archived)]
![image](https://github.com/alpinejs/alpine/assets/39895513/6c168264-b2db-4fcd-8a2f-279f098e2c05)

```html
<form x-data= "{ otherAddressOpen: false }"
            <details class="group/new_address [&_summary::-webkit-details-marker]:hidden"
                     x-data
                     :open="otherAddressOpen">
```

As the title states: the details open prop can't be bound.
I have tried with and without x-data on the details element.
The only way to control a details element open state is to have a `x-ref` and use `this.$refs.container.open = false;`

**Comment by ekwoka** [comment]
You are not supposed to directly manipulate the open property.

You are supposed to call `show()` and `close()` with the DOM api.

so this would more be

```html
<detail @other-opened.window="$el.close()">
```
